### ISSUE TITLE
How to use MapView with Tab View ?

### ISSUE THREAD

**Issue by kpbird** [JakeWharton/ActionBarSherlock#128]
Hello, 

I have three tab with Actionbar, 1st and 2nd tab has normal view, when user press 3rd tab it should display google map.

Kindly guide me how to achieve above functionality.

Thank You,

**Comment by JakeWharton** [comment]
Include the maps plugin, have your activity extend from `FragmentMapActivity`, and place the `MapView` in the fragment for the third tab. Take a look at `samples/plugins/` for a few examples of a map being used with the library.

For questions like this please use the forum at http://abs.io/forum

**Comment by ajmalsalim** [comment]
Hi,
I am using SlidingMenu along with ABS. So as usual I have 2 fragments. One is MenuFragment and other is ContentFragment. In ContentFragment, I would like to add two tabs in which a TextView in one Tab and a MapView in other Tab. I have tried many solutions but none of them are working as expected. 

I cannot extend FragmentMapActivity as I have extended SlidingFragmentActivity :P

I have tried this: http://xrigau.wordpress.com/2012/03/15/using-an-actionbar-in-your-application-with-actionbarsherlock/ and it worked !

But now when I click on the other fragments and come back to this fragment, both the layout disappeared.  :(
Any help will be really appreciated :)
